Dockerfile

```diff
@@ -1,12 +1,15 @@
-FROM node:20.5-alpine as node
+FROM docker.io/node:20.5-alpine as node
```

Comment:
Can you elaborate on this? Is there any benefit on specifically defining the registry, or is this a personal preference?

Reply:
I usually work with a lot of different registries and just got used to specifying it. There's no technical need for this. You are currently relying on this:

https://github.com/containers/shortnames/blob/0edfebac23765df6c34d19bcbcb2b6e4c65b56fe/shortnames.conf#L125

Reply:
Let's keep it as it was then.